cfpb/cfgov-refresh
# Bureau Structure: Resize on expandable expand/collapse | Set margin nesting.

Fixes https://[ghe]/CFGOV/platform/issues/3528

## Changes

- Re-add `initializeExpandables` from https://github.com/cfpb/cfgov-refresh/pull/4365 and call `heightChange` method when expandables expand/collapse.
- Scope margin in mobile view to only the first-level nodes.

## Testing

1. Pull branch, `gulp clean && gulp build`.
2. Visit http://localhost:8000/about-us/the-bureau/bureau-structure/
3. Resize the window size down to mobile view.
4. Scroll down the page and click the expandables note the bureau structure container height changes when they are expanded and collapsed.
5. Also note that margin on groupings are left-aligned.

## Screenshots

Before:

<img width="555" alt="Screen Shot 2019-08-28 at 9 58 06 AM" src="https://user-images.githubusercontent.com/704760/63863085-a4874680-c97b-11e9-92cc-57492b0ce319.png">



After:

<img width="553" alt="Screen Shot 2019-08-28 at 10 00 48 AM" src="https://user-images.githubusercontent.com/704760/63862585-cc29df00-c97a-11e9-8ee1-f943802c33be.png">


## Notes

- The event listeners for the expandables were originally removed when leaving mobile, however, there is a check in `heightChange` for whether the page is in the mobile size or not, so for simplicity I've left the event listeners attached at all window sizes.
